### DISCUSSION TITLE
Unblock all websites glitch

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#225] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Ok, the last one I made got patched, so lets try this again

Install geogebra classic from chrome webstore
Open it, obviously
Click the three (BIG) lines in the top right
hit sign in
Close the bigger window and make the smaller one fullscreen
Hit forgot password
In the bottom, there should be a small button that says license, click it
Scroll down until about halfway until you find this line : If you would like to help with developing GeoGebra please find GeoGebra's source code here: https://github.com/geogebra/geogebra.
Click the link in the line
It should now open a github page
in the search bar, type google and click all github
On the first result with a google logo, click the small button that says see topic
on the right margin with the links there should be a www.google.com link, so click it
Now, it should open a google page in which you can search and do anything with. (No blockers, no screen watchers like dyknow)

Sorry its a bit long

**Comment by PickleIsDaBest** [comment]
Great! It works!

**Comment by bartholemuew** [comment]
Can't install extensions.

**Reply by ghost** [reply]
oof, sorry about that
